recirq/quantum_chess/circuit_transformer_test.py

```diff
@@ -33,59 +33,64 @@
 @pytest.mark.parametrize('device',
                          (cirq.google.Sycamore23, cirq.google.Sycamore))
 def test_single_qubit_ops(device):
-    transformer = ct.ConnectivityHeuristicCircuitTransformer(device)
     c = cirq.Circuit(cirq.X(a1), cirq.X(a2), cirq.X(a3))
-    transformer.qubit_mapping(c)
-    c = transformer.transform(c)
-    device.validate_circuit(c)
+    t = ct.ConnectivityHeuristicCircuitTransformer(device)
+    device.validate_circuit(t.transform(c))
+    t = ct.DynamicLookAheadHeuristicCircuitTransformer(device)
+    device.validate_circuit(t.transform(c))
 
 
 @pytest.mark.parametrize('device',
                          (cirq.google.Sycamore23, cirq.google.Sycamore))
 def test_single_qubit_with_two_qubits(device):
```

Comment:
What does single_qubit_with_two_qubits mean?  Can we rename this to be more descriptive?

Reply:
renamed!